openrouteservice/src/main/java/heigit/ors/util/gpxUtil/PtType.java

```diff
@@ -0,0 +1,146 @@
+
+package heigit.ors.util.gpxUtil;
+
+import javax.xml.bind.annotation.*;
```

Comment:
We should avoid importing all items from the package to keep things clean

Reply:
Yes please, let's think lean!